projects/aws/eks-a-admin-image/Makefile

```diff
@@ -37,28 +37,49 @@ $(PACKER):
 	curl --silent -L https://releases.hashicorp.com/packer/$(PACKER_VERSION)/packer_$(PACKER_VERSION)_$(OS)_amd64.zip > packer.zip
 	unzip packer.zip -d $(@D)
 	rm packer.zip
-	$@ init .	
 
-$(FULL_OUTPUT_DIR):
-	mkdir -p $@
+$(FULL_OUTPUT_DIR)-%: IMAGE_TYPE=$*
+$(FULL_OUTPUT_DIR)-%:
+	mkdir -p $(FULL_OUTPUT_DIR)/$(IMAGE_TYPE)
 
 .PHONY: build-raw-image
 build-raw-image: build-ami export-to-raw
 
 .PHONY: build-ami
+build-ami: init
 build-ami: export PKR_VAR_eks-a-version = $(EKSA_VERSION)
 build-ami: export PKR_VAR_eks-a-release-manifest-url = $(EKSA_RELEASE_MANIFEST_URL)
 build-ami: export PKR_VAR_build-version = $(shell echo $(GIT_HASH) | cut -c1-7)
 build-ami: export PKR_VAR_manifest-output = $(MANIFEST_OUTPUT)
 build-ami: KUBECTL_URL?=$(shell ./build/get_kubectl_url.sh)
 build-ami: export PKR_VAR_kubectl-url = $(KUBECTL_URL)
-build-ami: $(PACKER) $(FULL_OUTPUT_DIR)
-	$(PACKER) build snow-admin.pkr.hcl
+build-ami: $(PACKER) $(FULL_OUTPUT_DIR)-ami
+	$(PACKER) build -only='*.amazon-ebs.*' eks-a-admin.pkr.hcl
 
 .PHONY: export-to-raw
 export-to-raw:
 	build/export-ami-to-s3.sh $(MANIFEST_OUTPUT) raw $(EXPORT_AMI_DST) $(LATEST_AMI_S3_URL)
 
+.PHONY: build-ova
+build-ova: init
```

Comment:
I prefer to put the prereqs all together. Specific reason you broke this out instead of putting it on line 74?

Reply:
No reason, I'll lump together as you suggest.